apps/app/src/app/pages/(components)/components/(tabs)/tabs.preview.ts

```diff
@@ -102,122 +102,128 @@ export class TabsPreviewComponent {}
 
 export const defaultCode = `
 import { Component } from '@angular/core';
+import { HlmBadgeDirective } from '@spartan-ng/ui-badge-helm';
+import { HlmButtonDirective } from '@spartan-ng/ui-button-helm';
 import {
-  BrnTabsComponent,
-  BrnTabsContentDirective,
-  BrnTabsListComponent,
-  BrnTabsTriggerDirective
-} from '@spartan-ng/ui-tabs-brain';
-import { HlmTabsContentDirective, HlmTabsListDirective, HlmTabsTriggerDirective } from '@spartan-ng/ui-tabs-helm';
-import {
-  HlmCardContentDirective,
-  HlmCardDescriptionDirective,
-  HlmCardDirective,
-  HlmCardFooterDirective, HlmCardHeaderDirective, HlmCardTitleDirective
+	HlmCardContentDirective,
+	HlmCardDescriptionDirective,
+	HlmCardDirective,
+	HlmCardFooterDirective,
+	HlmCardHeaderDirective,
+	HlmCardTitleDirective,
 } from '@spartan-ng/ui-card-helm';
-import { HlmLabelDirective } from '@spartan-ng/ui-label-helm';
 import { HlmInputDirective } from '@spartan-ng/ui-input-helm';
-import { HlmButtonDirective } from '@spartan-ng/ui-button-helm';
-import { HlmBadgeDirective } from '@spartan-ng/ui-badge-helm';
+import { HlmLabelDirective } from '@spartan-ng/ui-label-helm';
+import {
+	BrnTabsComponent,
+	BrnTabsContentDirective,
+	BrnTabsListComponent,
+	BrnTabsTriggerDirective,
+} from '@spartan-ng/ui-tabs-brain';
+import { HlmTabsContentDirective, HlmTabsListDirective, HlmTabsTriggerDirective } from '@spartan-ng/ui-tabs-helm';
 
 @Component({
-  selector: 'spartan-tabs-preview',
-  standalone: true,
-  imports: [
-    BrnTabsComponent,
-    BrnTabsListComponent,
-    BrnTabsTriggerDirective,
-    BrnTabsContentDirective,
+	selector: 'spartan-tabs-preview',
+	standalone: true,
+	imports: [
+		BrnTabsComponent,
+		BrnTabsListComponent,
+		BrnTabsTriggerDirective,
```

Comment:
Which of these are now covered by their Hlm counterparts? I think BrnTabsTriggerDirective can be removed